package.json

```diff
@@ -6,5 +6,8 @@
   "devDependencies": {
     "prettier": "^2.8.4",
     "prettier-plugin-solidity": "^1.1.3"
```

Comment:
truffle-assertions doesn't seem to be in the package.json, maybe try installing it again so it appears here. add to the dev dependencies `npm i -D truffle-assertions`

Reply:
oki done! thankuu